src/DbStructureModel.cpp

```diff
@@ -49,6 +50,9 @@ QVariant DbStructureModel::data(const QModelIndex& index, int role) const
         // For the display role and the browsable branch of the tree we want to show the column name including the schema name if necessary (i.e.
         // for schemata != "main"). For the normal structure branch of the tree we don't want to add the schema name because it's already obvious from
         // the position of the item in the tree.
+        if(index.column() == ColumnName && data(index.sibling(index.row(), ColumnObjectType), Qt::EditRole).toString() =="schema" ){
+            return item->text(index.column()) + " []"; // ?hn - remainder to add fileinfo later
```

Comment:
Scotty-sorry. You're commenting outdated changes. 
take a look on: Revert "rename "database" to "schema" internally" Please.

Reply:
Yeah.  GitHub is showing a little "[Outdated]" tag next to the code window.  I have no idea why it's showing people the old code, rather than the latest.  Doesn't seem useful. :frowning: